backend.go

```diff
@@ -5,6 +5,8 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/go-cleanhttp"
```

Comment:
Please run `goimports`

Reply:
Updated in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/142/commits/246c3f85023b0bcfd46ebc3f667658b274e74cf5

Reply:
It looks like one of the commits reverted your update, can we get this fixed once more?

Reply:
Updated in [e1765a3](https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/142/commits/e1765a3788f49f37fbf426d7ad8025613496d7a5)